index.html

```diff
@@ -66,26 +73,29 @@ <h2>I'm Jay</h2>
     </div>
   </section>
   <section class="works_container">
-    <div class="works_title">
+    <div id="works_title">
       <h2 class="name_title">My Recent Works</h2>
       <span class="line"></span>
     </div>
     <div class="cards">
       <div class="card1">
         <img class="gymimg" src="./Images/Gym.svg" alt="Gym image" />
-        <h3 class="card1_title">Multi-Post Stories</h3>
-        <p class="card1_paragraph">A daily section of privately personalized reads; no accounts or signups required.
-          This has been the industry's standard dummy text ever since the 1500s, when an
-          unknown printer took a standard dummy text.
-        </p>
-        <ul class="buttons1">
-          <li><button class="li_buttons1" type="button">css</button></li>
-          <li><button class="li_buttons1" type="button">html</button></li>
-          <li><button class="li_buttons1" type="button">bootstrap</button></li>
-          <li><button class="li_buttons1" type="button">ruby</button></li>
-        </ul>
-        <button class="card1button" type="button">See Project</button>
+        <div class="details_gyming">
+          <h3 class="card1_title">Multi-Post Stories</h3>
+          <p class="card1_paragraph">A daily section of privately personalized reads; no accounts or signups required.
+            This has been the industry's standard dummy text ever since the 1500s, when an
+            unknown printer took a standard dummy text.
+          </p>
+          <ul class="buttons1">
+            <li><button class="li_buttons1" type="button">css</button></li>
+            <li><button class="li_buttons1" type="button">html</button></li>
+            <li><button class="li_buttons1" type="button">bootstrap</button></li>
+            <li><button class="li_buttons1" type="button">ruby</button></li>
+          </ul>
```

Comment:
- [ ] Please note that your first project card is currently not visible. Kindly check your code and fix it 👍🏽  

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/186535406-0e334f7c-77a8-431b-b83f-b7d21e15e905.png)  |![image](https://user-images.githubusercontent.com/63473584/186535478-9872c8f9-321e-4516-b9b4-9c02ec96b482.png)  |